post-artifact/README.md

```diff
@@ -0,0 +1,93 @@
+# Post artifact as a comment [![Test PR post artifact](https://github.com/CDCgov/cfa-actions/actions/workflows/test-post-artifact.yml/badge.svg)](https://github.com/CDCgov/cfa-actions/actions/workflows/test-post-artifact.yml)
+
+## Inputs
+
+| Field         | Description                                                                                                                             | Required | Default        |
+|---------------|-----------------------------------------------------------------------------------------------------------------------------------------|----------|----------------|
+| `gh-token`      | The GitHub token to use for the API calls. | true | - |
+| `artifact-name` | Artifact name as in the `actions/upload-artifact` step. | false    | `artifact`     |
+| `message`       | Message template to be posted in the PR. The message should include a placeholder for { artifact-url }. Optionally, the { artifact-name } placeholder can be used to include the artifact name in the message. | false    | `'Thank you for your contribution ${{ github.actor }} :rocket:! Your { artifact-name } is ready for download :point_right: [here]({ artifact-url }) :point_left:!'` |
+| `python`        | The path to the Python executable. | false    | `'python'`       |
```

Comment:
Do you definitely want this to be configurable? Imagine it might be easier for action maintenance always to use the first `python` on `PATH`?

Reply:
Some systems may not have `python` but `python3`. I see this in the CI running `rocker/tidyverse` that has `python3` defined but not `python`.